cluster-up/cluster/kind-k8s-sriov-1.17.0/README.md

```diff
@@ -41,4 +42,32 @@ export KIND_VERSION="0.9.0"
 export KUBECTL_PATH="/usr/bin/kubectl"
 ```
 This allows users to test or use custom images / different kind versions before making them official.
-See https://github.com/kubernetes-sigs/kind/releases for details about node images according to the kind version.
\ No newline at end of file
+See https://github.com/kubernetes-sigs/kind/releases for details about node images according to the kind version.
+
+## Running multi sriov clusters locally
+Kubevirtci sriov provider supports running two clusters side by side with few known limitations.
+
+General considerations:
+
+- A sriov PF must be available for each cluster.
+In order to achieve that, there are two options:
+1. Assign just one PF for each cluster by using `export PF_COUNT_PER_NODE=1` (this is the default value).
```

Comment:
Yes, but its just a suggestion on how to run it locally, and its the default as one already, and even with #570 you will still have it 1, but you will do it for every worker if i understand right.
Anyhow its just a suggestion (for local use only), which i can remove if needed, since its the default already.

Reply:
once you merge the migration, u can update it that it will do it per worker, wdyt?
so the user will need to distrbute his PFs, between the cluster
where each cluster takes W*P
where W = num of workers
P = PF per worker (PF_COUNT_PER_NODE)

wdyt?

Reply:
This line is confusing, it made me though that PF_COUNT_PER_NODE control how many PF's each cluster will get instead of node
I think we need to change to reflect what the behavior would be:
Assign just one PF for each **node on each cluster **by using `export PF_COUNT_PER_NODE=1` (this is the default value).

As you may know eventually a SRIOV setup will require total of two PF's, one for each worker node to enable SRIOV live migration testing.
In case there are two cluster running on a CI node, then we would need total of 4 PF's 
2 for each cluster where each worker node will get one.
If this is how its going to work its fine by me.

Reply:
Migration is not yet supported, this is why i didn't update it when i created it (3 weeks ago at least, and even more since its a stripped mode of the one that had sriov-operator), and the variable name says its per NODE.
This is why i suggest to update it once the migration is merged and active on CI, not before that.
Currently NODE === Cluster because there is only one node per cluster that we support on sriov.
The right thing to be done is to update it only after migration updates this behavior, not before that,
as we look on the current state of the master.

Of course i know that, this is what i also tried to explain above,
and this is how it will work indeed, and still PF_COUNT_PER_NODE will be left as 1,
even with migration, because its one per node (this is what we agreed when we added this env var).

I will update it just to make the pipeline faster, but please consider next time that we should look om the current state of master, as we usually do.